lib/actions/ssh.coffee

```diff
@@ -98,11 +78,20 @@ module.exports =
 			.then ({ username, uuid, containerId, proxyUrl }) ->
 				throw new Error('Did not find running application container') if not containerId?
 				Promise.try ->
+					sshProxyCommand = ''
+					proxyConfig = global.PROXY_CONFIG
+					if proxyConfig
+						{ proxyAuth } = proxyConfig
+						proxyHost = "-p #{proxyConfig.host}:#{proxyConfig.port}"
+						proxyAuth = if proxyAuth then "-P #{proxyAuth}" else ''
+						proxytunnelCommand = "proxytunnel #{proxyHost} #{proxyAuth} -d %h:%p"
+						sshProxyCommand = "-o ProxyCommand='#{proxytunnelCommand}'"
```

Comment:
With this, it's impossible to use an HTTP proxy with the CLI unless you also proxy SSH too. That means if you use a proxy at all, you can't SSH unless you install proxytunnel.

That's a bit annoying, as I expect most people using proxies use them for HTTP(S) only, not for 100% of the traffic from their machines. Anybody who isn't on a network with SSH blocked doesn't need this extra hassle. Could we warn and try anyway, instead of letting `ssh` itself fail here? We could detect if `proxytunnel` is in PATH, and do this if so, but otherwise just print a warning:

> [WARN] Proxy configuration set, but proxytunnel is not in your $PATH. Connecting with SSH directly...'

That would also give a clearer explanation for people who don't read the instructions and fail to install proxytunnel/set it up correctly when they do need it.